google/resource_compute_backend_bucket.go

```diff
@@ -16,40 +29,38 @@ func resourceComputeBackendBucket() *schema.Resource {
 		Delete: resourceComputeBackendBucketDelete,
 
 		Importer: &schema.ResourceImporter{
-			State: schema.ImportStatePassthrough,
+			State: resourceComputeBackendBucketImport,
 		},
 
 		Schema: map[string]*schema.Schema{
-			"name": &schema.Schema{
+			"bucket_name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"name": {
 				Type:         schema.TypeString,
 				Required:     true,
 				ForceNew:     true,
-				ValidateFunc: validateGCPName,
+				ValidateFunc: validateRegexp(`^(?:[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)$`),
 			},
-
-			"bucket_name": &schema.Schema{
-				Type:     schema.TypeString,
-				Required: true,
-			},
-
-			"description": &schema.Schema{
+			"description": {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
-
-			"enable_cdn": &schema.Schema{
+			"enable_cdn": {
 				Type:     schema.TypeBool,
 				Optional: true,
-				Default:  false,
```

Comment:
Should we be worried about this Default getting removed?

Reply:
It's `Default: false` which as far as I know acts exactly the same as if there was no default at all, since `d.GetOk("enable_cdn")` would return (false, false) in both cases.

Reply:
What about the diff, would you get a pertual diff like the one below if you don't specify it?
```
"false" -> ""
  ^               ^
API            config

I am not sure what is the behavior for boolean.

Reply:
Good question! I just checked and it doesn't perpetual diff (and `terraform show` does say "false" for it)